src/Servers/HttpSys/src/RequestProcessing/ClientCertLoader.Log.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.HttpSys
+{
+    internal partial class ClientCertLoader
+    {
+        private static class Log
+        {
+            private static readonly Action<ILogger, Exception?> _channelBindingMissing =
+                LoggerMessage.Define(LogLevel.Error, LoggerEventIds.ChannelBindingMissing, "GetChannelBindingFromTls");
```

Comment:
This message could use some work (I know its not related to this change)

Reply:
Happy to update this PR with a more specific message for this if you have a suggestion.